lms/templates/experiments/dashboard_metadata.html

```diff
@@ -0,0 +1,17 @@
+<%page expression_filter="h"/>
+<%!
+from openedx.core.djangolib.js_utils import dump_js_escaped_json
+%>
+
+<%page args="course_prices"/>
+
+<%
+dashboard_metadata = {}
+for course in course_prices:
+    course_string = str(course)
+    dashboard_metadata[course_string] = course_prices[course]
```

Comment:
Assuming we will be adding to dashboard_metadata in the future, I recommend setting dashboard_metadata = { course_prices = course_prices } so we don't have to modify its structure later on, and to avoid unnecessary iteration over course_prices.

If we know absolutely that we will never add to dashboard_metadata ever, I would suggest renaming course_prices to dashboard_course_prices and including that instead, also to avoid unnecessary iteration over course_prices.

Reply:
Good suggestion. I do need to iterate here since the course id must be cast to a string somewhere and I thought doing that in the template made the most sense, but I can find a way to structure this better.

Reply:
I'm surprised course ID has to be cast as a string, what is it otherwise? If you're going to do any casting or data formatting, I'd do it when you def get_course_prices, since you're already iterating over there.